assets/credentials.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: cloudcredential.openshift.io/v1
+kind: CredentialsRequest
+metadata:
+  name: openshift-openstack-cinder-csi-driver
+  namespace: openshift-cloud-credential-operator
+spec:
+  secretRef:
+    name: openstack-cloud-credentials
+    namespace: openshift-openstack-cinder-csi-driver
+  providerSpec:
+    apiVersion: cloudcredential.openshift.io/v1
+    kind: OpenStackProviderSpec
```

Comment:
Recently we moved the CredentialsRequest creation to CVO. See https://github.com/openshift/cluster-storage-operator/blob/master/manifests/03_credentials_request_openstack.yaml

Reply:
done